ui/src/components/role/AddRole.js

```diff
@@ -293,10 +305,15 @@ class AddRole extends React.Component {
         let role = this.state.role;
         role.name = roleName;
         role.reviewEnabled = this.state.reviewEnabled;
+        role.auditEnabled = this.state.role.auditEnabled;
```

Comment:
let role = this.state.role doesn't do deep copy and there for its the same comment as the above.

Reply:
thanks, see comment above about using immer lib